### PR TITLE
Make the Vagrantfile.example fancier

### PR DESCRIPTION
**Pull request by jeremycline**
Signed-off-by: Jeremy Cline <jeremy@jcline.org>

### CODE REVIEW

**Comment by sayanchowdhury** [comment]
:+1:

File: Vagrantfile.example [hunk 1]
```diff
@@ -8,24 +8,65 @@
...
+ # ``vagrant-hostmanager``
+ if Vagrant.has_plugin?("vagrant-hostmanager")
+     config.hostmanager.enabled = true
+     config.hostmanager.manage_host = true
+ end
+
+ # Vagrant can share the source directory using rsync, NFS, or SSHFS (with the vagrant-sshfs
+ # plugin). Consult the Vagrant documentation if you do not want to use SSHFS.
+ config.vm.synced_folder ".", "/vagrant", disabled: true
+ config.vm.synced_folder ".", "/vagrant", type: "sshfs", sshfs_opts_append: "-o nonempty"
```
Comment (bowlofeggs) [inline Vagrantfile.example:33]: Why don't we go ahead and switch this to ```/home/vagrant/bodhi``` or something? I've been annoyed at working in ```/vagrant```…

File: Vagrantfile.example [hunk 2]
```diff
@@ -8,24 +8,65 @@
...
+
+ # Create the bodhi dev box
+ config.vm.define "bodhi" do |bodhi|
+    bodhi.vm.host_name = "bodhi-dev.example.com"
+
+    bodhi.vm.provider :libvirt do |domain|
+        # Season to taste
+        domain.cpus = 4
+        domain.graphics_type = "spice"
+        domain.memory = 4096
```
Comment (bowlofeggs) [inline Vagrantfile.example:61]: Maybe leave the comment that was here about the tests using a lot of RAM.

**Review by bowlofeggs — APPROVED** [review]
Two suggestions - take 'em or leave 'em. Thanks!

File: Vagrantfile.example (hunk 1 above)
Comment (jeremycline) [reply Vagrantfile.example:33]: I wanted to, but it involves changing all the Ansible stuff that expects that path explicitly. I figured it'd be best to do in a follow-up PR

**Comment by bowlofeggs** [comment]
On Thu, 2017-01-05 at 06:31 -0800, Jeremy Cline wrote:
> I wanted to, but it involves changing all the Ansible stuff that
> expects that path explicitly. I figured it'd be best to do in a
> follow-up PR

Oh right, good call.

File: Vagrantfile.example (hunk 1 above)
Comment (bowlofeggs) [reply Vagrantfile.example:33]: I'm going to make that follow-up PR now, to move ```/vagrant``` to ```/home/vagrant/bodhi```.

Comment (bowlofeggs) [reply Vagrantfile.example:33]: This PR moves the code to ```~/bodhi```: https://github.com/fedora-infra/bodhi/pull/1209

**Pull request merged**